### PR TITLE
Update links to repo inside docs

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Fix] Update and add links to repo source code from within docs. 
    - Recent repo refactor has invalidated some links to interactive diagram
    - As we static diagram code is now added into repo, hyperlinks also added where relevant.
- [Fix] Update references to npm run compile script to reference static or dynamic version where relevant.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ To start making your own custom diagram, follow the simple setup guide to start
```
Comment (JulianGoh5) [inline docs/README.md:31]: Noticed that there's two separate README files, one at the root repo level and this within `docs/README.md`. Not sure how you want to approach this, maybe could consolidate the two? The one at the repo level is now quite outdated

https://github.com/ClearCalcs/custom-diagram-boilerplate/blob/main/README.md

File: docs/README.md [hunk 2]
```diff
@@ -84,17 +84,17 @@ The Test Runner will also update automatically with your code changes while it's
 Once happy with changes, compile into a single html file at `output/index.html`.
 
 ```
-npm run-script compile
+npm run compile-interactive
```
Comment (JulianGoh5) [inline docs/README.md:87]: Just a thought on adding context to what the underlying logic is doing here, might be worth adding a sentence or so somewhere pointing out that this `npm run compile-interactive` ties into the `package.json`

Comment (JulianGoh5) [reply docs/README.md:87]: Ah nvm, I see some references already exist in `interactive-diagram-core.md` and `static-diagram-core.md`

**Review by JulianGoh5 — APPROVED** [review]
Links look great 👌

File: docs/README.md (hunk 1 above)
Comment (JulianGoh5) [reply docs/README.md:31]: Oh, you're already addressing this in the next one 👌 - https://github.com/ClearCalcs/custom-diagram-boilerplate/pull/33

File: docs/README.md (hunk 2 above)
Comment (michaellisitsa) [reply docs/README.md:87]: It's a Getting started guide not a tutorial, and expectation that developers know where npm scripts come from.

**Pull request merged**